deps.macos/20-libajantv2.zsh

```diff
@@ -34,10 +34,15 @@ config() {
 
   args=(
     ${cmake_flags}
-    -DAJA_BUILD_OPENSOURCE=ON
-    -DAJA_BUILD_APPS=OFF
+    -DAJANTV2_DISABLE_DEMOS=ON
+    -DAJANTV2_DISABLE_DRIVER=ON
+    -DAJANTV2_DISABLE_TESTS=ON
+    -DAJANTV2_DISABLE_TOOLS=ON
+    -DAJANTV2_DISABLE_PLUGINS=ON
     -DAJA_INSTALL_SOURCES=OFF
     -DAJA_INSTALL_HEADERS=ON
+    -DAJA_INSTALL_MISC=OFF
+    -DAJA_INSTALL_CMAKE=OFF
```

Comment:
Add the `:BOOL` type hint here as well - we haven't added them to existing build scripts consistently, but when modifying them we might as well fix that while we're at it.

Reply:
I did not see those type hints used in the other .zsh build scripts so I wasn't sure if it was desired. I only saw them used in the PowerShell scripts.

Reply:
Added the type hints to the macOS libajantv2 script.

Reply:
> I did not see those type hints used in the other .zsh build scripts so I wasn't sure if it was desired. I only saw them used in the PowerShell scripts.

Yep that's fine, as I said that's just a nice to have so we don't have to modify the file yet another time just for that (because at some point we'll need to add them to all .zsh files as well).